tox.ini

```diff
@@ -119,16 +121,16 @@ commands = isort -w 120 -rc luigi test examples bin
 # Build documentation using sphinx.
 # Call this using `tox -e docs`.
 deps =
-  sqlalchemy
-  boto3
-  Sphinx>=1.4.4,<1.5
-  sphinx_rtd_theme
-  enum34>1.1.0
-  azure-storage
-  prometheus-client==0.5.0
+    sqlalchemy
+    boto3
+    Sphinx>=1.4.4,<1.5
+    sphinx_rtd_theme
+    enum34>1.1.0
+    azure-storage
+    prometheus-client==0.5.0
 commands =
-    # build API docs
+# build API docs
     sphinx-apidoc -o doc/api -T luigi --separate
 
-    # build HTML docs
-    sphinx-build -W -b html -d {envtmpdir}/doctrees doc doc/_build/html
+# build HTML docs
+    sphinx-build -W -b html -d {envtmpdir}/doctrees doc doc/_build/html
```

Comment:
could you add back the end-of-file new-line?

Reply:
Done